scripts/globals/spells/trust/monberaux.lua

```diff
@@ -14,16 +14,30 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    -- TODO: Summon: I am Doctor Monberaux. My services are available for any affliction.
     -- TODO: Summon (1 dose of Final Elixir ready): I received a vial of a most valuable medicine. It should prove useful in times of emergency.
     -- TODO: Summon (2 doses of Final Elixir ready): I received two vials of a most valuable medicine. They should prove useful in times of emergency.
     xi.trust.message(mob, xi.trust.message_offset.SPAWN)
 
-    -- local healingMoveCooldown = 10
+    local healingMoveCooldown = 3
 
     -- Tends to be particular about which potions to use. Seems to favor healing for just the
     -- right amount of HP instead of defaulting to the highest-rank potion.
-    -- mob:addSimpleGambit(ai.t.PARTY, ai.c.HP_MISSING, 500, ai.r.MS, ai.s.SPECIFIC, 4236, healingMoveCooldown) -- Max Potion
+    mob:addMod(xi.mod.MPP, -80)
+    mob:addMod(xi.mod.HPP, 50)
+    mob:addSimpleGambit(ai.t.CASTER, ai.c.MPP_LT, 25, ai.r.MS, ai.s.SPECIFIC, 4254, healingMoveCooldown)
```

Comment:
No comment here, what move is this?

Reply:
Skill 4254 is Mix: Dry Ether Concoction, which he uses to restore MP on retail for himself and others. I added the caster condition because otherwise he kept spamming it on jobs with no MP.